lib/kubernetes-deploy/kubernetes_resource/memcached.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class Memcached < KubernetesResource
```

Comment:
Can we follow up with a PR refactoring the existing resources to follow the same more idiomatic patterns? I think consistency in this codebase is important.

Reply:
https://github.com/Shopify/kubernetes-deploy/issues/184

I will take care of doing that.